### ISSUE TITLE
Please Patch the latest versions of EDGE both x86 & x64

### ISSUE THREAD

**Issue by darkalexis1990** [Blaukovitch/Microsoft-EDGE-Windows-7-CRACK#9]
@Blaukovitch Please kindly patch the latest versions of Edge x86 and x64 ->
Version 121.0.2277.98: February 1, 2024 is the latest as these lines are written

also don't forget to make the reg patch to make edge the default browser as you did on chrome.

https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel

i hope also to have positive news from you as for spotify.

thank you :)

**Comment by darkalexis1990** [comment]
@Blaukovitch is there a more quick way to contact with you? maybe discord? is there a way to DM you private?

**Comment by darkalexis1990** [comment]
![image](https://github.com/Blaukovitch/Microsoft-EDGE-Windows-7-CRACK/assets/107497874/9a2ab9a7-6e85-4632-9828-f4a9c4153695)

you must update edge ,its unusable i get this message all the time please update the latest versions!

**Comment by Blaukovitch** [comment]
@darkalexis1990, yes, I read messages from time to time - I'm just **very busy**, so I may not reply for a while, but I'm aware of the issue anyway. 
         Yes, of course I will patch the latest version of **EDGE** browser, but I want to do it with a universal patcher, so as not to waste extra time - EDGE has a bit more work to patch than Chrome. Also, from the **last one** - I managed to run WebGPU on Windows 7 via DirectX 11.

**Comment by darkalexis1990** [comment]
@Blaukovitch try to patch x86 latest spotify when you have time .maybe that solve the problem with the broken links.also do you have a e-mail or is there a more fast way to direct contact you?

**Comment by Blaukovitch** [comment]
@darkalexis1990, at the moment I rarely appear on communication channels because of a large amount of work. Probably not :/

**Comment by darkalexis1990** [comment]
@Blaukovitch  can you please check to patch the latest version of spotify when you have time? also check the link bug that i have posted here. thanks youre the best!

**Comment by Blaukovitch** [comment]
>  latest version of ### spotify x86

Latest version of **Spootify** have some news library's.

**Comment by darkalexis1990** [comment]
@Blaukovitch new version Version 123.0.2420.53: March 22, 2024
